### DISCUSSION TITLE
树莓派4b 8G按照移植rtthread教程卡在了最后一步

### DISCUSSION THREAD

**Discussion by ForsakenDelusion** [RT-Thread/rt-thread#9274] [Q&A]
```shell
U-Boot 2024.10-rc1-00182-g6becf9ba1ab8 (Aug 03 2024 - 16:15:37 +0800)

DRAM:  948 MiB (effective 7.9 GiB)
RPI 4 Model B (0xd03115)
Core:  212 devices, 16 uclasses, devicetree: board
MMC:   mmcnr@7e300000: 1, mmc@7e340000: 0
Loading Environment from FAT... OK
In:    serial,usbkbd
Out:   serial,vidconsole
Err:   serial,vidconsole
Net:   eth0: ethernet@7d580000

Hit any key to stop autoboot:  0
ethernet@7d580000 Waiting for PHY auto negotiation to complete.... done
BOOTP broadcast 1
BOOTP broadcast 2
*** Unhandled DHCP Option in OFFER/ACK: 125
*** Unhandled DHCP Option in OFFER/ACK: 125
DHCP client bound to address 192.168.1.9 (302 ms)
Using ethernet@7d580000 device
TFTP from server 192.168.1.6; our IP address is 192.168.1.9
Filename 'rtthread.bin'.
Load address: 0x208000
Loading: ##################################################  645.5 KiB
         769.5 KiB/s
done
Bytes transferred = 660976 (a15f0 hex)
## Starting application at 0x00208000 ...
```

尝试过使用文档内的uboot64.bin,但发现进如uboot显示的内存为3.9g,且依然会卡在这一步.所以自己编译了最新的uboot,仍然会卡在上图所示这一步.交叉编译器为gcc-arm-11.2-2022.02-mingw-w64-i686-aarch64-none-elf

**Comment by heartofrain** [comment]
I meat the same problem. Do you have any idea about it now?

**Reply by ForsakenDelusion** [reply]
I couldn't find a solution and eventually gave up on using the Raspberry Pi. If you've found a way to solve it, I hope you can share it with me.

**Reply by heartofrain** [reply]
Finally I used [32bit version](https://github.com/RT-Thread/rt-thread/tree/master/bsp/raspberry-pi/raspi4-32) instead. It works very well.

**Reply by Drtangbo** [reply]
> Finally I used [32bit version](https://github.com/RT-Thread/rt-thread/tree/master/bsp/raspberry-pi/raspi4-32) instead. It works very well.

can you share it with me，thanks！

**Reply by Drtangbo** [reply]
> I couldn't find a solution and eventually gave up on using the Raspberry Pi. If you've found a way to solve it, I hope you can share it with me.

Have you solved this problem？if so，i hope you can share it with me，thanks！
